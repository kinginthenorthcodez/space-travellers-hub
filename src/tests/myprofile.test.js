import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { Provider } from 'react-redux';
import MyProfile from '../components/myprofile';
import store from '../redux/configureStore';

// Dragons component tests
describe('Myprofile snapshot', () => {
  afterEach(cleanup);

  it('should render myprofile correctly', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <MyProfile />
      </Provider>,
    );
    expect(asFragment(<MyProfile />)).toMatchSnapshot();
  });
});
