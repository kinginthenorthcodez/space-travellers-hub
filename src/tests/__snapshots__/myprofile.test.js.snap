// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Myprofile snapshot should render myprofile correctly 1`] = `
<DocumentFragment>
  <hr
    class="line"
  />
  <div
    class="myProfileContainer"
  >
    <div
      class="profileGroupContainer"
    >
      <h3
        class="profileHeader"
      >
        My Missions
      </h3>
      <div
        class="profileGroup"
      >
        <p
          class="noEntry"
        >
          No missions reserved
        </p>
      </div>
    </div>
    <div
      class="profileGroupContainer"
    >
      <h3
        class="profileHeader"
      >
        My Rockets
      </h3>
      <div
        class="profileGroup"
      >
        <p
          class="noEntry"
        >
          No rockets reserved
        </p>
      </div>
    </div>
    <div
      class="profileGroupContainer"
    >
      <h3
        class="profileHeader"
      >
        My Dragons
      </h3>
      <div
        class="profileGroup"
      >
        <p
          class="noEntry"
        >
          No dragons reserved
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
